frontend/src/components/reports/Routine.js

```diff
@@ -56,23 +56,9 @@ export const RoutineReportsMenu = {
       icon: IbmWatsonNaturalLanguageUnderstanding,
       SideNavMenuItem: [
         {
-          link:
-            config.serverBaseUrl +
-            "/Report?type=indicator&report=activityReportByTest",
+          link:"/RoutineReport?type=indicator&report=activityReportByTest",
           label: <FormattedMessage id="sideNav.label.bytesttype" />,
         },
-        {
-          link:
-            config.serverBaseUrl +
-            "/Report?type=indicator&report=activityReportByPanel",
-          label: <FormattedMessage id="sideNav.label.bypaneltype" />,
```

Comment:
@sherrif10 you dont have to remove the menu items. you only had to refactor the report componenet

Reply:
https://github.com/I-TECH-UW/OpenELIS-Global-2/assets/48015736/f2df03b9-a7a6-41cf-900d-acd6af1f2448

Restored it back onto three components.